authui/src/authgear.ts

```diff
@@ -309,6 +309,10 @@ window.api.onLoad(() => {
     }
   }
 
-  connect();
+  // Avoid websocket connection unless it is necessary.
+  if (window.location.pathname.startsWith("/verify_identity")) {
```

Comment:
I remembered we discussed on this and decided to always keep a WebSocket connection for simplicity, is there any problem with that?

Reply:
We can remove this once we have implemented dynamic `subscribe` / `unsubscribe`.